.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,10 +1,31 @@
-## What changes were proposed in this pull request?
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+1. If this is your first time, please read our contributor guidelines: https://spark.apache.org/contributing.html
+2. Ensure you have added or ran the appropriate tests for your PR: https://spark.apache.org/developer-tools.html
+3. If the PR is unfinished, add `[WIP]` in your PR title.
```

Comment:
how about adding an example like;
```
3. If the PR is unfinished, add `[WIP]` in your PR title, e.g., [WIP][SPARK-XXXX] Your PR title...
```

Reply:
+1. Let me collect more feedback and address them later in batch.

Reply:
yea, its ok. thx